src/components/AddItem/AddItem.js

```diff
@@ -5,11 +5,23 @@ import NavBar from '../NavBar/NavBar';
 const AddItem = (props) => {
   const [groceryItem, setGroceryItem] = useState('');
   const [howSoon, setHowSoon] = useState(null);
+  const [purchDate, setPurchDate] = useState(null);
+
+  const oneday = 60 * 60 * 24 * 1000;
+
+  const workOutNextPurchDate = (howSoon) => {
+    const howManyDaysInSeconds = howSoon * oneday;
```

Comment:
Could possibly add precision to variable names. oneday is really dayInMilliseconds and howManyDaysInSeconds is really purchaseIntervalInMilliseconds (though I get the latter is a bit long so might need some abbreviation)

Reply:
Agreed here! I guess you could call it purchaseIntervalInMs? Since knowledge of time in MS is common enough?